src/sentry/quotas/redis.py

```diff
@@ -81,8 +81,12 @@ def get_quotas(self, project, key=None):
             ))
         return tuple(results)
 
-    def get_redis_key(self, key, timestamp, interval):
-        return '{}:{}:{}'.format(self.namespace, key, int(timestamp // interval))
+    def get_redis_key(self, key, timestamp, interval, shift):
```

Comment:
… and, as it turns out, the code path that used this introduced method actually never worked anyway, since it the queries were routed to the wrong shard.